packages/editor/src/components/post-taxonomies/hierarchical-term-selector.js

```diff
@@ -407,8 +407,7 @@ export function HierarchicalTermSelector( { slug } ) {
 		<Flex direction="column" gap="4">
 			{ showFilter && (
 				<TextControl
-					// TODO: Switch to `true` (40px size) if possible
-					__next40pxDefaultSize={ false }
+					__next40pxDefaultSize
```

Comment:
The category filter that appears in the post sidebar when there are many categories.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/2fd1c84b-dbda-4ada-b867-7668b75f1fc7" alt="Search categories, before" width="273">|<img src="https://github.com/user-attachments/assets/256608cf-aeb2-478a-be4c-5469802f7126" alt="Search categories, after" width="273">|


May want to switch to a SearchControl (#64593).